investors/views.py

```diff
@@ -202,3 +211,64 @@ def perform_destroy(self, instance):
             extra={"saved_id": instance.pk, "by_user": self.request.user.pk},
         )
         super().perform_destroy(instance)
+
+class ViewedStartupPagination(pagination.PageNumberPagination):
+    """
+    Pagination class for recently viewed startups.
+    Default page size is 10, can be overridden via ?limit query parameter.
+    """
+    page_size = 10
+    page_size_query_param = "limit"
```

Comment:
The pagination parameter is named `limit` instead of the DRF standard `page_size`. Rename to `page_size` for DRF convention, or document the deviation.
```
page_size_query_param = "page_size"  # or document the use of "limit"
```